Gemfile.lock

```diff
@@ -382,7 +382,7 @@ GEM
     i18n (1.8.5)
       concurrent-ruby (~> 1.0)
     ice_nine (0.11.2)
-    image_processing (1.10.3)
+    image_processing (1.12.1)
```

Comment:
used by `carrierwave`, a bunch of enhancements https://github.com/janko/image_processing/blob/master/CHANGELOG.md but nothing meaningful to report